tableauserverclient/server/endpoint/tasks_endpoint.py

```diff
@@ -9,26 +9,41 @@
 class Tasks(Endpoint):
     @property
     def baseurl(self):
-        return "{0}/sites/{1}/tasks/extractRefreshes".format(self.parent_srv.baseurl,
-                                                             self.parent_srv.site_id)
+        return "{0}/sites/{1}/tasks".format(self.parent_srv.baseurl,
+                                            self.parent_srv.site_id)
 
-    @api(version='2.6')
-    def get(self, req_options=None):
-        logger.info('Querying all tasks for the site')
-        url = self.baseurl
+    def __normalize_task_type(self, task_type):
+        """
+            The word for extract refresh used in API URL is "extractRefreshes".
+            It is different than the tag "extractRefresh" used in the request body.
+        """
+        if task_type == TaskItem.Type.ExtractRefresh:
+            return '{}es'.format(task_type)
+        else:
+            return task_type
+
+    @api(version='3.8')
```

Comment:
We don't want to change the version, because this endpoint works for extractRefreshes for version 2.6+. I think the best way for now is to call assert_at_least_version() from the parent server class if tasktype is materalizeviews.

Reply:
Thanks a lot for pointing it out! Changed it as advised.